src/pages/UsersPage.js

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react";
+import { List } from "../components/List";
+
+export const UsersPage = () => {
+  const [data, setData] = useState([]);
+
+  const [inputValue, setInputValue] = useState("");
+
+  const fetchData = async (queryString = "/") => {
+    const response = await fetch(
+      `https://jsonplaceholder.typicode.com/users${queryString}`
+    );
+    const usersData = await response.json();
+    setData(usersData);
+  };
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const columnNames = [
+    { value: "User ID", keyName: "id" },
+    { value: "Name", keyName: "name" },
+    { value: "Username", keyName: "username" },
+  ];
+
+  const handleChange = (event) => {
+    setInputValue(event.target.value);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    if (inputValue) {
```

Comment:
Instead of if else here, use a variable using ternary operator.

```javascript
   const queryString = inputValue.length ? `?username=${inputValue}` : "";
   fetchData(queryString);

```

Reply:
Done